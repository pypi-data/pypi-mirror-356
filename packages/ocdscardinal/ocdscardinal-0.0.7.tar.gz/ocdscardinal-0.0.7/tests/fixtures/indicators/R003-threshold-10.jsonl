{"ocid":"10","tender":{"tenderPeriod":{"startDate":"2000-01-10T18:00:00Z","endDate":"2000-01-20T17:59:59Z"}}}
{"ocid":"11","tender":{"tenderPeriod":{"startDate":"2000-01-10T18:00:00Z","endDate":"2000-01-21T17:59:59Z"}}}
