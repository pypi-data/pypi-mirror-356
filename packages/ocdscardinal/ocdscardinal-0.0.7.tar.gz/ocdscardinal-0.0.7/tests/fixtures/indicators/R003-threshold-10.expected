{"OCID":{"10":{"R003":1.0}}}